dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java

```diff
@@ -127,84 +146,73 @@ synchronized MonitorStats getMonitorStatsNoCache()  {
         return monitorStats;
     }
 
-
+    /**
+     * Determines if the database server is healthy by executing a simple query.
+     *
+     * @return If the database server is healthy, returns {@code true}.
+     */
     boolean isDBHealthy()  {
-
-
             return Try.of(()->
-                            new DotConnect().setSQL("SELECT count(*) as count FROM (SELECT 1 FROM dot_cluster LIMIT 1) AS t")
+                            new DotConnect().setSQL("SELECT 1 as count")
                     .loadInt("count"))
                     .onFailure(e->Logger.warnAndDebug(MonitorHelper.class, "unable to connect to db:" + e.getMessage(),e))
                     .getOrElse(0) > 0;
-
-
     }
 
-
-    boolean isIndexHealthy(final String index) {
-
-        SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
-        searchSourceBuilder.query(QueryBuilders.matchAllQuery());
-        searchSourceBuilder.size(0);
-        searchSourceBuilder.timeout(TimeValue
-                .timeValueMillis(INDEX_OPERATIONS_TIMEOUT_IN_MS));
-        searchSourceBuilder.fetchSource(new String[]{"inode"}, null);
-        SearchRequest searchRequest = new SearchRequest();
-        searchRequest.source(searchSourceBuilder);
-        searchRequest.indices(index);
-
-        long totalHits = Try.of(()->
-                RestHighLevelClientProvider
-                        .getInstance()
-                        .getClient()
-                        .search(searchRequest,RequestOptions.DEFAULT)
-                        .getHits()
-                        .getTotalHits()
-                        .value)
-                .onFailure(e->Logger.warnAndDebug(MonitorHelper.class, "unable to connect to index:" + e.getMessage(),e))
-                .getOrElse(0L);
-
-        return totalHits > 0;
-
+    /**
+     * Determines if dotCMS can connect to Elasticsearch by pinging the server.
+     *
+     * @return If dotCMS can connect to Elasticsearch, returns {@code true}.
+     */
+    boolean canConnectToES() {
+        try {
+            RestHighLevelClientProvider.getInstance().getClient().ping(RequestOptions.DEFAULT);
```

Comment:
this returns a boolean - let's check for it being true

Reply:
Done!